### DISCUSSION TITLE
Got error in pythoncode while running wifi-img-streamer.c

### DISCUSSION THREAD

**Discussion by thebikerholic** [bitcraze/discussions#123] [Q&A]
i have runned the code [wifi-img-streamer.c](https://github.com/bitcraze/aideck-gap8-examples/blob/master/examples/other/wifi-img-streamer/wifi-img-streamer.c) but i got the error in the p-rograme while ruunning it i have attached the error.
![Screenshot from 2022-07-04 02-16-07](https://user-images.githubusercontent.com/101856934/177078541-825d5df7-0fb1-4126-a0d8-6da1bfa6dd80.png)

**Comment by evoggy** [comment]
Hi!

Do you get this the first time you connect, or do you get it after first connection, disconnecting and the connecting again? It seems as there's no protection in the python script for jumping into the middle of a stream.

**Reply by thebikerholic** [reply]
i got this error at the first time itself while connecting to crazyfly.

**Reply by evoggy** [reply]
Ok, and the script you're using is a copy of the one we supply? What the error message means is that it's not receiving enough data from the deck. Have you modified any of the AI deck firmware (ESP32 or GAP8)?

**Reply by thebikerholic** [reply]
i am surely not clear about please help me to rectify this problem

**Reply by evoggy** [reply]
Is the script you're running the same as [this one](https://github.com/bitcraze/aideck-gap8-examples/blob/master/examples/other/wifi-img-streamer/opencv-viewer.py)? And the firmware you're running on the GAP8, does it have any modifications or is it the same as the [example here](https://github.com/bitcraze/aideck-gap8-examples/blob/master/examples/other/wifi-img-streamer/wifi-img-streamer.c)?
